src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const confirmedTabId = tabs.some(({ id }) => id === selectedTabId)
+    ? selectedTabId
+    : tabs[0].id;
+  const selectedTabContent = tabs
+    .find(({ id }) => id === confirmedTabId).content;
```

Comment:
Same here like with the title. It's better to create a variable for `selectedTab` and then get all the needed properties from it

Reply:
Could do much more, but the tests here are rather specific